### PR TITLE
Bluewash

### PR DESCRIPTION
**Pull request by mathieucarbou**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -236,17 +236,6 @@
...
-    <license>
-      <name>The Apache Software License, Version 2.0</name>
-      <url>http://www.ehcache.org/about/license</url>
-      <distribution>repo</distribution>
-      <comments>The license is the standard wording from the Apache license.
-                Terracotta, Inc. is the copyright owner.
-      </comments>
-    </license>
-  </licenses>
-
```
Comment (mathieucarbou) [inline pom.xml:249]: parent POMs do not need to have that - this is a section for projects

**Review by chrisdennis — COMMENTED** [review]
Should the pom itself get an Apache header and copyright declaration?

**Comment by mathieucarbou** [comment]
> Should the pom itself get an Apache header and copyright declaration?

I don't know, and this comment applies to many projects actually, so if the answer is yes, then we have to go back through all other projects to verify that also...

**Review by prasantakumar82 — APPROVED** [review]

**Comment by prasantakumar82** [comment]
> > Should the pom itself get an Apache header and copyright declaration?
> 
> I don't know, and this comment applies to many projects actually, so if the answer is yes, then we have to go back through all other projects to verify that also...

I have added copyright header in one pom.xml file and it did not have before.

**Comment by mathieucarbou** [comment]
> > > Should the pom itself get an Apache header and copyright declaration?
> > 
> > 
> > I don't know, and this comment applies to many projects actually, so if the answer is yes, then we have to go back through all other projects to verify that also...
> 
> I have added copyright header in one pom.xml file and it did not have before.

yes but this one is a parent pom. Copyright should be added in libraries or applications. What's the use for a copyright inside a parent pom ?

**Pull request merged**